controllers/rhmi/bootstrapReconciler.go

```diff
@@ -727,11 +727,15 @@ func generateSecret(length int) string {
 
 func (r *Reconciler) processQuota(installation *rhmiv1alpha1.RHMI, namespace string,
 	installationQuota *quota.Quota, serverClient k8sclient.Client) error {
+	var err error
```

Comment:
As agreed I think we can update the logic here to also create the initial config map here that will list the components and their target utilization. 
Also, I think some unit tests around this logic and cm map creation/no reconciliation on 1+ runs would be really good.

Reply:
Added this in the recent commit :)